oasst-shared/oasst_shared/schemas/inference.py

```diff
@@ -41,8 +41,11 @@ def __init__(self, **data):
         data["uname_processor"] = platform.uname().processor
         data["cpu_count_physical"] = psutil.cpu_count(logical=False)
         data["cpu_count_logical"] = psutil.cpu_count(logical=True)
-        data["cpu_freq_max"] = psutil.cpu_freq().max
-        data["cpu_freq_min"] = psutil.cpu_freq().min
+        # psutil.cpu_freq()is None for conatainers running on an Apple silicon host
+        # https://github.com/giampaolo/psutil/issues/1892
+        cpu_freq = psutil.cpu_freq()
```

Comment:
Until 1892 is fixed
if platform.machine() == 'arm64' can be used to update cpu_freq with 0 or None